### PR TITLE
Fix IE check on FastBoot

### LINKED ISSUE #529: Cannot read property 'userAgent' of undefined on FastBoot
Hi there! 

I encountered a problem using the power-select with Ember FastBoot. 

```
There was an error running your app in fastboot. More info about the error:
 TypeError: Cannot read property 'userAgent' of undefined
    at Module.callback (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:88373:47)
    at Module.exports (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:119:34)
    at Module.build (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:174:10)
    at findModule (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:223:9)
    at Module.reify (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:151:22)
    at Module.build (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:173:10)
    at findModule (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:223:9)
    at requireModule (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:26:12)
    at [object Object]._extractDefaultExport (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:92577:20)
    at [object Object].resolveOther (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:92312:32)
    at [object Object].superWrapper (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:32386:22)
    at [object Object].exports.default._emberRuntimeSystemObject.default.extend.resolve (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:16075:35)
    at resolve (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:12553:36)
    at Object.Registry.resolve (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:12014:21)
    at Object.Registry.resolve (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:12018:55)
    at factoryFor (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:11509:28)
    at Object.Container.lookupFactory (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:11375:14)
    at [object Object].exports.default._emberMetalMixin.Mixin.create._lookupFactory (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:42750:33)
    at [object Object].exports.default._emberRuntimeSystemObject.default.extend.componentFor (/Users/josemarluedke/Projects/manager/dist/fastboot/vendor.js:50757:2

```

I suspect that the problem is coming from https://github.com/cibernox/ember-power-select/blob/master/addon/components/power-select-multiple/trigger.js#L7.

### PR DESCRIPTION
**Pull request by josemarluedke**
This fixes #529.

### CODE REVIEW

File: addon/components/power-select-multiple/trigger.js [hunk 1]
```diff
@@ -11,8 +11,18 @@ import { isBlank } from 'ember-utils';
...
-const isIE = ua.indexOf('MSIE ') > -1 || ua.indexOf('Trident/') > -1;
+
+const isFastBoot = () => {
+  return (typeof process !== 'undefined' && process.env && process.env.EMBER_CLI_FASTBOOT);
+};
+
+const isIE = () => {
+  if (isFastBoot()) { return false; }
+
+  const ua = self.window ? self.window.navigator.userAgent : '';
```
Comment (cibernox) [inline addon/components/power-select-multiple/trigger.js]: Maybe it is easier to just do
```js
const ua = self.window && self.window.navigator ? self.window.navigator.userAgent : '';
```

Comment (josemarluedke) [inline addon/components/power-select-multiple/trigger.js]: 👍 Changed!

**Comment by josemarluedke** [comment]
@cibernox changes applied. Ready for merge.

**Comment by cibernox** [comment]
The master branch is broken for other reasons. I'll cherry pick this commit onto the 0.10 branch tonight

**Comment by cibernox** [comment]
I've cherry-picked this change into 0.10.11.

**Pull request merged**